controllers/swiftproxy_controller.go

```diff
@@ -122,35 +122,89 @@ func (r *SwiftProxyReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	serviceLabels := swiftproxy.Labels()
 
 	// Create a Service and endpoints for the proxy
-	var swiftPorts = map[endpoint.Endpoint]endpoint.Data{
-		endpoint.EndpointAdmin: endpoint.Data{
+	var swiftPorts = map[service.Endpoint]endpoint.Data{
+		service.EndpointAdmin: {
```

Comment:
we do not create admin endpoint for other services, is the admin endpoint required for swift?

Reply:
Good point - no, it's no longer required, this should work fine without it.